examples/adaptive-cards/index.html

```diff
@@ -2,10 +2,12 @@
 
 <html lang="en">
         <title>ComposeUI Adapative cards POC App</title>
+        <link rel="stylesheet" type="text/css" href="styles.css">
     </head>
 
     <body>
         <div id="container"></div>
+        <script type="text/javascript" src="https://unpkg.com/markdown-it/dist/markdown-it.min.js"></script>
```

Comment:
CDNs can time out and be blocked on the firewall.
Could you change the consumption of markdown-it to be through an installed NPM package?